<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>796</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Airport Schedule</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QGroupBox" name="gb_airport_data">
      <property name="title">
       <string>Выбор данных о рейсах</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lb_airport">
         <property name="text">
          <string>Аэропорт:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_airport"/>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_flight_dirrection">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string/>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="rb_departure">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>Вылет</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_arrival">
            <property name="text">
             <string>Прилет</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="de_flight_date">
         <property name="maximumDate">
          <date>
           <year>2017</year>
           <month>9</month>
           <day>14</day>
          </date>
         </property>
         <property name="minimumDate">
          <date>
           <year>2016</year>
           <month>8</month>
           <day>15</day>
          </date>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_flight_info">
         <property name="text">
          <string>Список рейсов</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="gb_stats">
      <property name="title">
       <string>Статистика</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pb_airport_stats">
         <property name="text">
          <string>Загруженность аэропорта</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="vs_stats">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <spacer name="vs_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" rowspan="3">
     <widget class="QTableView" name="tv_flight_info"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>База данных</string>
    </property>
    <addaction name="act_db_info"/>
    <addaction name="act_db_conn"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="act_db_info">
   <property name="text">
    <string>Доступ к БД</string>
   </property>
  </action>
  <action name="act_db_conn">
   <property name="text">
    <string>Подключение</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
